src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/profile_2017_03_09/test_vm_actions.py

```diff
@@ -68,6 +74,9 @@ def test_generate_specfied_ssh_key_files(self):
         self.assertTrue(os.path.isfile(public_key_file4 + '.private'))
         self.assertTrue(os.path.isfile(public_key_file4))
 
+        # delete temporary directory and its files
+        shutil.rmtree(temp_dir_name)
```

Comment:
Throw it in `self.AddCleanup` - https://docs.python.org/3/library/unittest.html?highlight=unittest%20addcleanup#unittest.TestCase.addCleanup

Reply:
I think I have addressed your comments, thanks for the feedback.